pkg/KubeArmorOperator/manifest/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Can we add a single generated manifest file to the `deployments` directory like we have for other controllers? Helps in easier deployments :sweat_smile: